apps/condo/domains/common/utils/serverSchema/index.js

```diff
@@ -21,6 +21,24 @@ const logger = getLogger('common/utils/serverSchema.js')
 // Tested on tickets export for 24755 tickets: without knex  71220.902ms, with knex: 16094.841ms )
 // TODO(zuch): find out how to make 1 request for 1. and 2.
 
+function logTooManyReturnedIfRequired (tooManyReturnedMemory, allObjects, { limit, functionName, schemaName, data }) {
+    if (tooManyReturnedMemory.length < 5) return  // trying to notify only if have not notidied yet. Max 4 times
+
+    let realLimit = limit  // notify on: limit, limix * 10, limit * 100, limit * 1000
+    for (let i = 0; i < tooManyReturnedMemory.length; i++) realLimit *= 10
```

Comment:
I think it would be better to just count these limits on file load and then use here, like

const TOO_MANY_RETURNED_LIMITS = [10, 10*10, 10*100, 10*1000] 

and then use in logic using your function

This way we wont have to count these on every function call